packages/@aws-cdk/aws-ecs-patterns/test/fargate/scheduled-fargate-task.test.ts

```diff
@@ -94,6 +102,7 @@ test('Can create a scheduled Fargate Task - with optional props', () => {
       cpu: 2,
       environment: { TRIGGER: 'CloudWatch Events' },
     },
+    eventRole: role,
```

Comment:
This test needs to also verify that the correct role is being referenced in the right place(s) in the stack. This role should be referenced in the validation part of the test as well. 

Also - a note about unit tests: Make sure to merge from master before making edits. This module has been migrated to the new Assertions unit testing framework since this PR was opened.